examples/function_memory_game/src/components/game_status_board.rs

```diff
@@ -0,0 +1,39 @@
+use crate::constant::Status;
+use yew::prelude::*;
+use yew::{function_component, html, Properties};
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    pub status: Status,
+    pub sec_past: u32,
+    pub on_reset: Callback<()>,
+}
+
+#[function_component(GameStatusBoard)]
+pub fn game_status_board(props: &Props) -> Html {
+    let get_content = {
+        let onclick = props.on_reset.reform(move |e: MouseEvent| {
+            e.stop_propagation();
+            e.prevent_default();
+        });
+
+        match props.status {
+            Status::Ready => html! {
+                <span >{"Ready"}</span>
+            },
+            Status::Playing => html! {
+                <span >{"Playing"}</span>
+            },
+            Status::Passed => html! {
+                <a onclick={onclick}>{"Play again"}</a>
```

Comment:
```suggestion
                <a {onclick}>{"Play again"}</a>
```

Reply:
Could you make this a `<button>` ?
It has no reason to be link as it does not redirect anywhere.
Also take a look at the lint we have regarding assistive technologies. We have quite a few yew users that use them, so making examples friendly to them is worth it.

Reply:
sure